### PR TITLE
Website landing page

### PR DESCRIPTION
**Pull request by vishwanath1004**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "ionicons": "^7.0.0",
     "jwt-decode": "^4.0.0",
     "network-service": "^0.0.1",
+    "ng-otp-input": "^2.0.5",
```
Comment (5Amogh) [inline package.json:42]: Is this needed for our portal landing page? if not can we please remove this?

File: src/app/core/constants/introPageConstant.ts
Comment (5Amogh) [inline src/app/core/constants/introPageConstant.ts:1]: do we need to use some form config or something so in future if they want to change the content of the portal?

File: src/app/intro/intro.page.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+          >
+            {{ section?.buttons?.label }}
+          </ion-button>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="footer">
+      <h1>Join us for every step towards education</h1>
```
Comment (5Amogh) [inline src/app/intro/intro.page.html:90]: if the adaptor wants to take this up then education would not come so in that if they have different statement then we should be able to insert them maybe via that above mentioned form config or something

File: src/app/core/constants/introPageConstant.ts
Comment (vishwanath1004) [reply src/app/core/constants/introPageConstant.ts:1]: The design is not yet finalized hence i kept is as constant file once that's done then we can do that.

File: src/app/intro/intro.page.html (hunk 2 above)
Comment (vishwanath1004) [reply src/app/intro/intro.page.html:90]: Yeah will take this also from constant file as of now. Thanks for finding.

**Pull request merged**